test/test_plugin_loading_failures.py

```
@@ -0,0 +1,347 @@
+
+
+class TestErrorAggregation:
+    """Test that errors are properly recorded in error aggregator."""
+
+    def test_plugin_load_error_recorded(self, temp_plugin_dir, mock_managers):
+        """Plugin load errors should be recorded in error aggregator."""
+        from src.error_aggregator import get_error_aggregator
+
+        # Get the aggregator
+        aggregator = get_error_aggregator()
+        initial_count = len(aggregator._records)
```

Ruff (0.14.14)

[error] Local variable `initial_count` is assigned to but never used

Remove assignment to unused variable `initial_count`

(F841)